app/src/main/java/org/fossasia/openevent/general/utils/CurrencyUtils.kt

```diff
@@ -0,0 +1,24 @@
+package org.fossasia.openevent.general.utils
+
+import timber.log.Timber
+import java.util.*
+
+object CurrencyUtils {
+    private var currencyLocaleMap = HashMap<Currency, Locale>()
+
+    init {
+        for (locale in Locale.getAvailableLocales()) {
+            try {
+                val currency = Currency.getInstance(locale)
+                currencyLocaleMap.put(currency, locale)
+            } catch (e: IllegalArgumentException) {
+                Timber.d(e, "Failed!")
+            }
+        }
+    }
+
+    fun getCurrencySymbol(currencyCode: String?): String? {
+        val currency = Currency.getInstance(currencyCode)
+        return currency.getSymbol(currencyLocaleMap[currency])
```

Comment:
I think there is no need of creating a util class for this
 `currency.symbol` does the same thing what this entire util class is doing.

Reply:
Okay :+1:

Reply:
@nikit19 Updated PR